crates/era-cheatcodes/tests/foundry.toml

```diff
@@ -5,6 +5,6 @@ libs = ['lib']
 
 [rpc_endpoints]
 local = "${ERA_TEST_NODE_RPC_URL}"
-mainnet = "https://mainnet.era.zksync.io:443"
+mainnet = "${RPC_MAINNET}"
```

Comment:
Can we change this variable to be `local`? Just so its consistent with using era-test-node.

Reply:
Sure, I'll make the change . FYI this involves changing all references in tests (such as https://github.com/matter-labs/foundry-zksync/blob/eb2ae26710a124f3e66bf0cb52b41d335cf20d9b/crates/era-cheatcodes/tests/src/cheatcodes/Fork.t.sol#L25) to reference the "local" endpoint.